web-console/future-openshift/toolbars/toolbars.md

```diff
@@ -0,0 +1,49 @@
+# Toolbars
+
+There are various toolbars used across the OpenShift console.
+
+
+## Basic toolbars
+* The most basic toolbars in OpenShift have a textbox filter and an item count. The primary 'Create' button lives outside the toolbar in line with the page title.
+
+Example
+![deployments](img/deployments-today.png)
+
+## Toolbars with faceted filters
+* This type of toolbar is similar to the basic toolbar but adds faceted filters. The faceted filter and the textbox filter are separated by a vertical separator.
+* When filters are applied, the toolbar increases in height to show filter chips on the second row.
+* Pages with faceted filter toolbars may or may not have a 'Create' button.
+* Toolbars with faceted filters may have opinionated views. In these cases, filters are already applied when a user navigates to that page, and the filter chips are visible to show what is being filtered.
+* Faceted filters may allow users to filter on more than one attribute where necessary.
+
+Example with 'Create' button and opinionated view
+![pods](img/pods-today.png)
+
+Example without 'Create' button and no opinionated view
+![nodes](img/nodes-today.png)
+
+Faceted filter with multiple attributes
+![nodes2](img/nodes-future.png)
```

Comment:
Could the filters in the Select also convey the count for them (maybe right aligned)? Wasn't sure if that was technically possible. For example:

Role
[ ] Master     (5)
[ ] Worker     (2)

Reply:
Yes, good point! I think that would be really helpful. I thought PatternFly had this option but I don't see it in the documentation. I think it's feasible, but wonder if @rachael-phillips has any insight?

Reply:
Opened a PatternFly [issue](https://github.com/patternfly/patternfly-next/issues/2422) to get this added. In the meantime, I think we can merge this.